include/CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+cmake_minimum_required(VERSION 3.8.0)
+
+project(pcg VERSION 0.98.1 LANGUAGES CXX)
```

Comment:
Only the root `CMakeLists.txt` needs this. I also think having a separate `CMakeLists.txt` for the include directory for a header only library to be a bit overkill.

Reply:
Ditto.

Basically, `include` is the "root" directory for the project. If you read the previous comments in the thread, my suggestion was to have `include` under yet another directory such as `src`. We can discuss the name, perhaps have `include` under `pcg`, etc., but the most important thing is to have a separate project directory.

Having more than 1 `CMakeLists.txt` files does not hurt anyone. `README.md` can easily document what you need to do. If you want to install the headers, run
```bash
mkdir build
cd build
cmake ../include # or, cmake ../pcg, cmake ../src, whatever that folder would be
cmake --build . && cmake --build . --target install
```
If you want to also install sample files, do the super build:
```bash
mkdir build
cd build
cmake ../
cmake --build && cmake --build . --target install
```

At the end of the day, you simplify the logic.

Reply:
I guess I just find it weird since I've not seen any header only library separate them like that.

---

> We can discuss the name, perhaps have include under pcg, etc.

This should probably be raised as an issue and resolved before this PR is merged.

Reply:
Agreed.